pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -309,29 +309,29 @@ func (l *podNIC) newLibvirtSpecGenerator(domain *api.Domain) (LibvirtSpecGenerat
 
 func newMacvtapLibvirtSpecGenerator(iface *v1.Interface, domain *api.Domain) *MacvtapBindMechanism {
 	return &MacvtapBindMechanism{
-		iface:  iface,
-		domain: domain,
+		vmiSpecIface: iface,
+		domain:       domain,
```

Comment:
I expected this in commit https://github.com/kubevirt/kubevirt/pull/5880/commits/bf078f49fbcfdfa323d7c0df6c012c3c37900fc0 

Is there a reason for splitting this change in 2 commits (1 for renaming on podnic, another for the bindings) ?

Reply:
squashed.